pkg/config/db.go

```diff
@@ -43,6 +43,7 @@ type (
 		SessionReplicationRole        *SessionReplicationRole `toml:"session_replication_role"`
 		SharedBuffers                 *string                 `toml:"shared_buffers"`
 		StatementTimeout              *string                 `toml:"statement_timeout"`
+		TrackActivityQuerySize        *uint                   `toml:"track_activity_query_size"`
```

Comment:
This field needs to be a string because it can support "1MB" https://postgresqlco.nf/doc/en/param/track_activity_query_size/

Reply:
It doesn't it's an integer, the "1MB" is for when you set it's value to: 1048576

See:
https://www.postgresql.org/docs/17/runtime-config-statistics.html#GUC-TRACK-ACTIVITY-QUERY-SIZE

Reply:
Well.. that's the confusing part of their docs. I realised this when addin other `*_size` fields like https://postgresqlco.nf/doc/en/param/effective_cache_size/

You can run the following query to check that 1kB is indeed valid.

```sql
show track_activity_query_size;
```

Reply:
Good catch, updated accordingly both the cli and the api PR: https://github.com/supabase/infrastructure/pull/21241/files